sdk/include/aditof/sensor_definitions.h

```diff
@@ -57,6 +57,13 @@ struct SensorDetails {
     ConnectionType connectionType;
 };
 
+struct BufferInfo {
+    /**
+     * @brief Timestamp of the last requested frame
```

Comment:
"Timestamp of the data within the buffer" should be more appropriate

Reply:
Fixed.